src/routes/(console)/project-[region]-[project]/+layout.ts

```diff
@@ -32,23 +36,22 @@ export const load: LayoutLoad = async ({ params, depends }) => {
             });
         }
         await preferences.loadTeamPrefs(project.teamId);
+
+        // get values from saved stores if available!
+        const [rolesStoreValue, scopesStoreValue] = [get(rolesStore), get(scopesStore)];
+        const areBothAvailable = !!(rolesStoreValue.length && scopesStoreValue.length);
```

Comment:
also will create weird conditions, because potentially they are wrongly invalidated.

in short we should never need to call stores in those files.